samples/net/wifi/wifi_station/src/main.c

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
I guess `subsys/net/l2/wifi_shell.c` is a superset, right, we don't need a separate wifi_shell sample?

Reply:
You mean removing `wifi_shell` sample?

Reply:
Not quite, I don't see what extra this sample is adding that is not there in the `wifi_shell` sample? Except that `wifi_shell` automatically selects `wifi_mgmt` but I guess that is fine.

Reply:
I see. The whole idea is to have a `station` sample and not rely on the `wifi_shell.c` subsystem. I find it very difficult for new users to get Zephyr's working way without sample codes. Anyway, `station` sample is just duplicating, I can removed it. I'll wait more feedback.

Reply:
The commit log needs to change accordingly.